src/styles/_bootstrap_variables_mapping.scss

```diff
@@ -93,7 +93,7 @@
   // Links
   //
   // Style anchor elements.
-  --bs-link-color: #{$link-color};
+  --bs-link-color: white;
```

Comment:
I think better solution could be updating `link-color` what do you think?

Reply:
Agree